memcached.c

```diff
@@ -5972,12 +6108,20 @@ static int server_socket(const char *interface,
                 int per_thread_fd = c ? dup(sfd) : sfd;
                 dispatch_conn_new(per_thread_fd, conn_read,
                                   EV_READ | EV_PERSIST,
-                                  UDP_READ_BUFFER_SIZE, transport);
+                                  UDP_READ_BUFFER_SIZE, transport
+#ifdef TLS
+                                  , NULL
```

Comment:
think instead of #ifdef soup we can always have the extra parameter here.

but inside the function, use an UNUSED() macro or actually use the TLS bits.